src/webgl/p5.RendererGL.Retained.js

```diff
@@ -118,6 +118,22 @@ p5.RendererGL.prototype.drawBuffers = function(gId) {
   const gl = this.GL;
   const geometry = this.retainedMode.geometry[gId];
 
+  if (this._doFill) {
```

Comment:
Out of curiosity, whats the impact of moving this block of code up?

Reply:
Filling before drawing lines means that if fills and lines have the same z value, the lines end up on top. The main reason for this is to make drawing work a bit more like in 2D mode where lines are centered and thickness extends both into and out of the shape. This is already the order we use when rendering in immediate mode actually, so this just makes both modes consistent: https://github.com/processing/p5.js/blob/8c9430921f6f3db9360b5fb3e90dd76011877cf6/src/webgl/p5.RendererGL.Immediate.js#L197-L206